content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -29,13 +29,24 @@ For clarity, this guide defines the following terms:
 {{< link text="services" url="/docs/concepts/services-networking/service/" >}} within the cluster.
 Traffic routing is controlled by rules defined on the Ingress resource.
 
-```none
-    internet
-        |
-   [ Ingress ]
-   --|-----|--
-   [ Services ]
-```
+Here is a simple example where an Ingress sends all its traffic to one Service:
+{{< mermaid >}}
```

Comment:
I think the issue mentioned below is easily solve-able with a bit of CSS.

@sftim, I have [a commit for you on a branch in my fork](https://github.com/celestehorgan/kubernetes-website/commit/c6ec2b97aafd6573702a7d0a9543a341609bfe54) that I think solves this :)

Reply:
(Alternately, we could set the width of `.mermaid` to be a bit wider than the text's 80% width. Setting tables, images and diagrams a bit wider if you think that's a good idea is a standard book design practice – up to 90% would work well and be visually pleasing.)

Reply:
I cherry-picked that commit. I agree that `.mermaid` would work better darawn wider, maybe even 100% width.

Reply:
Hi. I looked at the latest changes. The scrollbar made me chuckle a bit.
I think I asked this before:
Is there a strong reason to dynamically render diagrams versus creating an SVG file?
Can you create SVG files from the Mermaid instructions in Markdown file or must
you use the Mermaid CLI? I am wondering if the instructions could be saved separately from
the content file. Right now, I don't see the value of dynamically generating the diagrams.

Reply:
If I get time, I'll see about making a SVG version that is similar to what's in the Mermaid diagram.